blueocean-core-js/src/js/NotFound.jsx

```diff
@@ -15,7 +15,7 @@ export default () => (
                 defaultValue: 'Page not found (404)',
             })}</h3>
             <div className="message">{translate('Not.found.message', { defaultValue: 'Jenkins could not find the page you were looking for. Check the URL for errors or press the back button.' })}</div>
-            <div className="actions"><Link to="/" className="btn btn-primary inverse">{translate('Open.dashboard', { defaultValue: 'Open Dashboard' })}</Link></div>
```

Comment:
This one did not break actually. The only test I saw that was selecting against a class such as `.btn  .btn-primary  .btn-secondary` was in the  "loadGerman" test. I fixed that issue in 6c3f889